roles/blockscout/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+user: ubuntu
+
+allow_blockscout_ssh: true
+allow_blockscout_p2p: true
+allow_blockscout_http: false
+
+username: "blockscout"
+users:
+- name: "{{ username }}"
+home: "/home/{{ username }}"
+
+MAIN_REPO_FETCH: "poanetwork"
+MOC_ADDRESS: "0xdd0bb0e2a1594240fed0c2f2c17c1e9ab4f87126"
+
+reboot_after_completion: false
+install_debug_tools: "no"
+install_firewall: false
+
+blockscout_version: "df6ec81b68"
```

Comment:
Is it better to use `master` here or it might break things?

Reply:
The build process is changed a bit from the time I wrote this code, so it wouldn't work with master right now. I'll update it to the current version. I think it's better to use exact version while BlockScout is under active development